base/tps/shared/conf/CS.cfg

```diff
@@ -2105,7 +2105,7 @@ preop.internaldb.data_ldif=/usr/share/pki/tps/conf/db.ldif,/usr/share/pki/tps/co
 preop.internaldb.index_ldif=/usr/share/pki/tps/conf/index.ldif
 preop.internaldb.ldif=/usr/share/pki/server/conf/database.ldif
 preop.internaldb.manager_ldif=/usr/share/pki/server/conf/manager.ldif
-preop.internaldb.post_ldif=/usr/share/pki/tps/conf/vlv.ldif,/usr/share/pki/tps/conf/vlvtasks.ldif
+preop.internaldb.post_ldif=/usr/share/pki/tps/conf/vlv.ldif,/usr/share/pki/tps/conf/vlvtasks.ldif,/usr/share/pki/tps/conf/auditors.ldif
```

Comment:
As discussed on IRC, files defined in this property will be imported during the initial installation as well, so the additional entry in db.ldif is sufficient, it's not necessary to add auditors.ldif.

Currently there is no database upgrade framework, so you might need to write a standalone program to add the LDAP entry into the database.